src/core/a-assets.js

```diff
@@ -163,18 +164,30 @@ function mediaElementLoaded (el) {
         resolve();
       }
     }
+
+    // Respect WebKit's user-gesture requirement for auto-playback of
+    // video containing audio. For more info:
+    // https://webkit.org/blog/6784/new-video-policies-for-ios/
+    window.addEventListener('click', handleFirstClick);
+    function handleFirstClick () {
+      try {
+        if (el.paused) { el.play(); }
```

Comment:
I haven't tested the differences between all the browsers, but to be on the safe side, I think this should be checking `el.autoplay` instead of just `el.getAttribute('autoplay')`, but I think it's fine